spike_main.py

```diff
@@ -162,8 +162,10 @@ def main ():
       # max_val = np.max(wave_form, axis=1)
       # wave_form = wave_form / max_val[:, None]
 
-      wave_form = get_sample_spikes ()
-      total_spikes += len(wave_form)
+      wave_form = []
+      for _ in range(30):
```

Comment:
@zerobfd you might want to change this number to adjust how many spikes you want to process at once.